pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/renderer/AmbulationValueProvider.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.dsm.model.elastic.export.tabular.renderer;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.model.elastic.export.tabular.FilterExportConfig;
+
+public class AmbulationValueProvider extends TextValueProvider {
```

Comment:
That seems like a very study (AT) specific thing. I wonder if we need other special behavior classes than that. Could we please add the study name to the class name
My two thoughts come up here:
1) What makes that question special
2) Are there any other studies that might have that problem/question type

Reply:
good question, I had thought this was a standard code.  I wonder if we need a framework for putting study-specific export behavior inside the packages for that study?